artemis/task_utils.py

```diff
@@ -9,6 +9,25 @@
 
 
 def get_target_host(task: Task) -> str:
+    """
+    Extract the target host identifier from a Karton task based on its type.
+
+    This function extracts different identifiers depending on the task type:
```

Comment:
IMO this paragraph is not needed, as it just describes the implementation. I would rather elaborate on the high-level purpose of the function - it takes the task and returns the host (domain/ip) it refers to